data/bytedance

```diff
@@ -25,6 +25,9 @@ zijiecdn.com
 zijiecdn.net
 zjcdn.com
 
+# Baike
+baike.com
```

Comment:
tracker detected in `i.snssdk.com`.
sample addr: https://i.snssdk.com/log/sentry/v2/api/slardar/batch/

Reply:
still this may be dangerous to add. need more samples.

Reply:
I found this domain in https://github.com/AdguardTeam/AdguardFilters/blob/6093e1217945343209cba918456eb38cd26da2df/SpywareFilter/sections/tracking_servers.txt#L16.

It might be safer to include `mcs.snssdk.com` only.